### PR TITLE
MINOR: [Dev] Add Gang Wu to CODEOWNERS for cpp/parquet and cpp/orc

### PR DESCRIPTION
**Pull request by wgtmac**

### CODE REVIEW

**Comment by wgtmac** [comment]
cc @pitrou

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -29,13 +29,14 @@
...
 /cpp/src/arrow/acero @westonpace
+/cpp/src/arrow/adapters/orc @wgtmac
 /cpp/src/arrow/dataset @westonpace
 /cpp/src/arrow/engine @westonpace
 /cpp/src/arrow/flight/ @lidavidm
 /cpp/src/arrow/util/async* @westonpace
 /cpp/src/arrow/util/future* @westonpace
 /cpp/src/arrow/util/thread* @westonpace
-/cpp/src/parquet @wjones127
+/cpp/src/parquet @wjones127 @wgtmac
```
Comment (pitrou) [inline .github/CODEOWNERS:39]: @wjones127 Do you still want to be listed here?

Comment (wgtmac) [reply .github/CODEOWNERS:39]: Feel free to directly edit this line. @wjones127 @pitrou

Comment (wjones127) [inline .github/CODEOWNERS:39]:
```suggestion
/cpp/src/parquet @wgtmac
```

**Review by wjones127 — APPROVED** [review]

**Review by kou — APPROVED** [review]
+1

**Pull request merged**